pkg/aws/helpers.go

```diff
@@ -568,3 +568,21 @@ func ValidateSubnetsCount(multiAZ bool, privateLink bool, subnetsInputCount int)
 
 	return nil
 }
+
+const (
+	singleAZCount = 1
+	MultiAZCount  = 3
+)
+
+func ValidateAvailabilityZonesCount(multiAZ bool, availabilityZonesCount int) error {
```

Comment:
This func and the constanst are not AWS specific. Maybe should be in a generic helper, or OCM helper

Reply:
Make sense, Moved to `pkg/ocm/helpers.go`.